### ISSUE TITLE
log file extension not supported by Github.

### ISSUE THREAD

**Issue by TZubiri** [rubberduck-vba/Rubberduck#2068]
Could we change it to txt so we can just drag and drop the file without changing the extension?
To make it easier for people who spend time filing bug reports.

**Comment by retailcoder** [comment]
This was brought up before. I fail to see a reason we'd need to hold on to a .log extension. Let's do this.

**Comment by daFreeMan** [comment]
The suggestion when I brought this up was to get Git to update to the 20th century and realize that `.log` is a legitimate text file extension, but I think this will be resolved more quickly by RD making the change.

**Comment by PeterMTaylor** [comment]
But GitHub does not support .txt files in my case only .zip

**Comment by retailcoder** [comment]
Ugh. Then the ultimate solution is to open the log file and paste the relevant entries into the issue?

**Comment by Hosch250** [comment]
> Unfortunately, we don’t support that file type. with a PNG, GIF, JPG, DOCX, PPTX, XLSX, TXT, PDF, or ZIP.

I don't see why it would only support .zip for you.

**Comment by PeterMTaylor** [comment]
Maybe not @retailcoder. Let me rephrase what I meant. 
When attempting to supply a .txt file log into an issue "ticket" eariler for #2063, I got the 
> Unfortunately, we don’t support that file type. with a PNG, GIF, JPG, DOCX, PPTX, XLSX, TXT, PDF, or ZIP.  

message so I used the Send to->Compressed File .ZIP thinking it would resolve this problem. 

When I saw other "tickets" in my inbox [yes I call this my @Duga spam! :) ] @TZubiri in #2058 did this successfully, we may ask how it was done.

I checked for reference [here](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests) how Github, this appears to be contraction in terms.
